best-practices/README.md

```diff
@@ -105,6 +107,7 @@ Rails
 [redirects]: http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.30
 [Spring binstubs]: https://github.com/sstephenson/rbenv/wiki/Understanding-binstubs
 [prevent tampering]: http://blog.bigbinary.com/2013/03/19/cookies-on-rails.html
+[class constant in association]: https://github.com/thoughtbot/guides/blob/master/style/rails/sample.rb
```

Comment:
Should this link to the relevant line? Or is that overkill since it is one class?